src/controllers/game.controller.ts

```diff
@@ -41,12 +37,16 @@ export async function start(
   const user = req.user;
   const game = await gameService.getById(id);
   if (game.hostId === user?._id) {
```

Comment:
In the start function, the condition if (game.hostId === user?._id) could be problematic if user is undefined.

Reply:
this user is request field, where user is set or exception is thrown, so there is user and it is defined